math-scala/src/main/scala/org/apache/mahout/drivers/MahoutOptionParser.scala

```diff
@@ -50,7 +50,7 @@ class MahoutOptionParser(programName: String) extends OptionParser[Map[String, A
         "(optional). Default: empty.")
     }
 
-    opt[String]('o', "output") required() action { (x, options) =>
+    opt[String]('o', "output") action { (x, options) =>
```

Comment:
You could add a default param to the function, like:

def parseIOOptions(numInputs: Int = 1, noOutput: Boolean = false)

Then a conditional around the requied(). This will work fine for everyone else with no change.